<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>951</width>
    <height>635</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout" stretch="1,4">
      <item>
       <layout class="QVBoxLayout" name="proLayout"/>
      </item>
      <item>
       <layout class="QVBoxLayout" name="picLayout"/>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>951</width>
     <height>20</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_file">
    <property name="title">
     <string>文件(&amp;F)</string>
    </property>
   </widget>
   <widget class="QMenu" name="menu_setting">
    <property name="title">
     <string>设置(&amp;S)</string>
    </property>
   </widget>
   <addaction name="menu_file"/>
   <addaction name="menu_setting"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="act_create_pro">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/res/icon/createpro.png</normaloff>:/res/icon/createpro.png</iconset>
   </property>
   <property name="text">
    <string>创建项目</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+N</string>
   </property>
   <property name="menuRole">
    <enum>QAction::NoRole</enum>
   </property>
  </action>
  <action name="act_open_pro">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/res/icon/openpro.png</normaloff>:/res/icon/openpro.png</iconset>
   </property>
   <property name="text">
    <string>打开项目</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
   <property name="menuRole">
    <enum>QAction::NoRole</enum>
   </property>
  </action>
  <action name="act_music">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/res/icon/music.png</normaloff>:/res/icon/music.png</iconset>
   </property>
   <property name="text">
    <string>背景音乐</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+M</string>
   </property>
   <property name="menuRole">
    <enum>QAction::NoRole</enum>
   </property>
  </action>
 </widget>
 <resources>
  <include location="res.qrc"/>
 </resources>
 <connections/>
</ui>
